packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1955,7 +1955,6 @@ class DiagnosticableNode<T extends Diagnosticable> extends DiagnosticsNode {
     String name,
     @required this.value,
     @required DiagnosticsTreeStyle style,
-    String emptyBodyDescription,
```

Comment:
cc @jacob314

Reply:
nice! the lint caught some dead code left over after refactoring.